stablehlo/reference/Ops.h

```diff
@@ -29,7 +29,7 @@ Tensor evalCeilOp(const Tensor &operand, Type resultType);
 Tensor evalConstantOp(ElementsAttr value);
 Tensor evalCosineOp(const Tensor &operand, Type resultType);
 Tensor evalFloorOp(const Tensor &operand, Type resultType);
-Tensor evalIotaOp(int64_t iotaDimension, Type resultType);
+Tensor evalIotaOp(const int64_t iotaDimension, Type resultType);
```

Comment:
I generally avoid const on primitive type when passed by value as  it does not provide any meaningful signal to the user: Just the fact that the argument is passed by value, provides guarantee that the actual argument is not going to change.